molgenis-core-ui/src/main/java/org/molgenis/ui/menumanager/MenuManagerServiceImpl.java

```diff
@@ -24,6 +25,7 @@
 import static java.util.Objects.requireNonNull;
 import static java.util.stream.Collectors.toList;
 
+@Component
```

Comment:
I think you can now remove the explicit java config @Bean method in MolgenisWebAppConfig